router/batchrouter/asyncdestinationmanager/snowpipestreaming/snowpipestreaming_test.go

```diff
@@ -43,6 +48,29 @@ func (m *mockAPI) GetStatus(_ context.Context, channelID string) (*model.StatusR
 	return m.getStatusOutputMap[channelID]()
 }
 
+type mockManager struct {
+	manager.Manager
+	throwSchemaErr bool
```

Comment:
Instead of a boolean, we can set the following error like `createSchemaErr`, `createTableErr`.

Reply:
Done